torchrl/envs/libs/vmas.py

```diff
@@ -627,22 +634,35 @@ class VmasEnv(VmasWrapper):
     Paper: https://arxiv.org/abs/2207.03530
 
     Args:
-        scenario (str or vmas.simulator.scenario.BaseScenario): the vmas environment to build.
+        scenario (str or vmas.simulator.scenario.BaseScenario): the vmas scenario to build.
+            Must be one of :attr:`.available_envs`. For a description and rendering of available scenarios see
+            https://github.com/proroklab/VectorizedMultiAgentSimulator?tab=readme-ov-file#main-scenarios
```

Comment:
Is there a way we can link that to a specific commit? I'm afraid this title changes one day and the link gets silently broken.
Also I would use the \`test \<link\>\`_ syntax

Reply:
If I link to a specific commit, users won't see new senarios

Reply:
see my comment below

Reply:
done